src/m365/purview/commands/retentionlabel/retentionlabel-add.spec.ts

```diff
@@ -56,6 +58,73 @@ describe(commands.RETENTIONLABEL_ADD, () => {
     dispositionReviewStages: []
   };
 
+  const eventTypeResponse = {
+    value: [
+      {
+        displayName: "Retention Event Type",
+        description: "",
+        createdDateTime: "2023-02-02T15:47:54Z",
+        lastModifiedDateTime: "2023-02-02T15:47:54Z",
+        id: "81fa91bd-66cd-4c6c-b0cb-71f37210dc74",
+        createdBy: {
+          user: {
+            id: "36155f4e-bdbd-4101-ba20-5e78f5fba9a9",
+            displayName: null
+          }
+        },
+        lastModifiedBy: {
+          user: {
+            id: "36155f4e-bdbd-4101-ba20-5e78f5fba9a9",
+            displayName: null
+          }
+        }
+      }
+    ]
+  };
+
+  const multipleEventTypesResponse = {
```

Comment:
See my other comment, I think this is a waste of code lines as event types should have unique names :-)